docs/dbus/pool.xml

```diff
@@ -0,0 +1,227 @@
+      -->
+      <arg name="result" type="(bs)" direction="out" />
+      <arg name="return_code" type="q" direction="out" />
+      <arg name="return_string" type="s" direction="out" />
+    </method>
+    <!-- Make a snapshot of a Stratis filesystem. -->
+    <method name="SnapshotFilesystem">
+      <!-- Object path of the origin filesystem. -->
+      <arg name="origin" type="o" direction="in" />
+      <!-- Name of the snapshot to be created. -->
+      <arg name="snapshot_name" type="s" direction="in" />
+      <!--
+        b: True if a new filesystem was created, otherwise false.
+        o: Object path of the snapshot.
+      -->
+      <arg name="result" type="(bo)" direction="out" />
+      <arg name="return_code" type="q" direction="out" />
+      <arg name="return_string" type="s" direction="out" />
+    </method>
+    <!-- Remove Clevis encryption from the devices in the pool. -->
+    <method name="UnbindClevis">
+      <!-- True if some Clevis bindings were removed, otherwise false. -->
+      <arg name="results" type="b" direction="out" />
+      <arg name="return_code" type="q" direction="out" />
+      <arg name="return_string" type="s" direction="out" />
+    </method>
+    <!-- Unbind a keyring passphrase from the devices in the pool. -->
+    <method name="UnbindKeyring">
+      <!-- True if some bindings were removed, otherwise false. -->
+      <arg name="results" type="b" direction="out" />
+      <arg name="return_code" type="q" direction="out" />
+      <arg name="return_string" type="s" direction="out" />
+    </method>
+    <!-- Total space allocated from the pool. -->
+    <property name="AllocatedSize" type="s" access="read" />
+    <!--
+      Indicates the current capabilities of the pool. Generally the pool has
+      full capability, but under certain circumstances stratisd may restrict
+      the actions that the pool can perform.
+    -->
+    <property name="AvailableActions" type="s" access="read" />
+    <!--
+      b: True if the pool is encrypted using Clevis.
+      b(ss): Clevis encryption configuration.
+         b: True if the Clevis configuration is consistent for all the
+	    block devices in the pool, otherwise false.
+         ss: Clevis configuration.
+	   s: Clevis pin.
+	   s: Pin-specfic Clevis configuration information.
+    -->
+    <property name="ClevisInfo" type="(b(b(ss)))" access="read" />
+    <!-- True if the pool is encrypted, otherwise false. -->
+    <property name="Encrypted" type="b" access="read">
+      <annotation name="org.freedesktop.DBus.Property.EmitsChangedSignal" value="const" />
+    </property>
+    <!-- True if the pool has a cache, otherwise false. -->
+    <property name="HasCache" type="b" access="read" />
+    <!--
+      b: True if the pool is encrypted using a key in the kernel keyring.
+      bs: Encryption information.
+         b: True if the kernel keyring is the same for all the block devices
+	    in the pool, otherwise false.
```

Comment:
This appears to be in the wrong order. `b(bs)` represents `StratisResult<Option<_>>`, not the other way around.